src/views/webviewController.ts

```diff
@@ -83,9 +83,7 @@ export default class WebviewController {
     connectionAttemptId: string
   ): Promise<void> => {
     try {
-      const connectionModel = this._connectionController
-        .parseNewConnection(rawConnectionModel as any);
-
+      const connectionModel = this._connectionController.parseNewConnection(rawConnectionModel);
```

Comment:
Should this be called `connectionInfo`? Otherwise it’s confusing, because this is a `ConnectionInfo`, not a `ConnectionModel`